src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -66,6 +68,24 @@ class Favourites extends Component {
         }
     }
 
+    onMouseOver(stockCode) {
```

Comment:
Might be easier just to use the event to get a reference to the favourite element like this:

```
    onMouseOver(e) {
        e.currentTarget.getElementsByClassName('favourite')[0].classList.add('hovered');
    }

    onMouseOut(e) {
        e.currentTarget.getElementsByClassName('favourite')[0].classList.remove('hovered');
    }
```